changelogs/fragments/101-plugin-param-ordering-note.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - Add a note about the ordering of positional and named parameter to the plugin page.
+bugfix:
+  - Make sure the positional and keyword parameter usage notes are also properly generated for lookup plugins.
```

Comment:
This is not a bugfix, but a new feature. Also please remove this feature from this PR since lookups work differently than filters and tests. (The one postional parameter is not a single parameter, but a list of all parameters passed.)